### PR TITLE
GH-8: Update tsconfig to be stricter and readme to be clearer

### PR DESCRIPTION
**Pull request by tamarayustian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,11 +37,12 @@ A repo built on ExpressJS and React with TypeScript
 
 ## Commit Practices
 
-1. Use the _command line_ for commits
+1. Run `yarn format` in the root directory
```
Comment (bhryck) [inline README.md:40]: what does this do @yuenyauu

Comment (yuenyauu) [reply README.md:40]: It formats all the relevant files based on prettier.config, so our codebase's formatting will be consistent

File: server/README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+## Installation
+
+1. Check out the first README at root directory if you haven't yet
+2. Run `yarn install` at `./server`
+
+## Usage
+
+1.  Whitelist your current IP address for our database access
+
```
Comment (yuenyauu) [inline server/README.md:13]: maybe remove the newline?

File: server/README.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+    - You can follow our tutorial [here](https://t3ch.super.site/guides/mongodb-setup-guide) or set up your own database
+
```
Comment (yuenyauu) [inline server/README.md:15]: this newline too

File: ui/README.md [hunk 4]
```diff
@@ -1,59 +1,29 @@
...
-
-### `yarn build`
-
-Builds the app for production to the `build` folder.<br /> It correctly bundles
-React in production mode and optimizes the build for the best performance.
+## Installation
 
-The build is minified and the filenames include the hashes.<br /> Your app is
-ready to be deployed!
+1. Check out the first README at root directory if you haven't yet
```
Comment (yuenyauu) [inline ui/README.md:8]: Check out the README, can remove the 'first' i think

**Review by yuenyauu — APPROVED** [review]

**Review by Midnite1212 — APPROVED** [review]

**Pull request merged**